web/src/people.ts

```diff
@@ -222,6 +222,22 @@ export function update_email(user_id: number, new_email: string): void {
     // still work correctly.
 }
 
+// A function that sorts Email according to the user's full name
+export function sort_emails(emails: string[], user_ids: number[]): (string | undefined)[] {
```

Comment:
I think it'd be better to be more explicit here by naming it something like `sort_emails_by_name`.